view/templates/widget/posted_date.tpl

```diff
@@ -31,10 +31,16 @@ function showHideDates() {
 		</li>
 		<li id="posted-date-selector-{{$y}}" class="tool posted-date-selector-months" style="display: none;">
 			<ul class="datebrowse-ul">
+				{{if $y|cat:$thisday >= $cutoffday}}
```

Comment:
What does this line do? I'm not familiar with the `cat` modifier.

Reply:
It concatenates the year in question to this month and day.  So if its less than the $cutoffday it does nothing.

Reply:
It wasn't showing any change or difference from messages.po after running, but then I deleted my clone and pulled a fresh clone and there were differences.  commits should be a bit easier now.

Reply:
Fixed the translation sting file

Reply:
Ohhh that's what I'm doing wrong.  gonna fix

Reply:
I feel like I just keep making this worse and worse.  How can I do this?  I tried using the github desktop to revert the messages.po en-us and I feel it screwed everything up.

Reply:
The revert doesn't show in the commit list in this PR. Ideally, you should squash all the commits in this branch when you rebase it once more onto 8ec3f17, and amend the resulting single commit to remove the changes that were wrongly introduced.

To squash several commits into one during rebase, you have to mark all the subsequent commits beyond the first as "fixup".

Reply:
https://friendica.myportal.social/photo/1393519529647e39cf55d08034697118-0.png
So I did this in fork, Got this error 
error: cannot 'fixup' without a previous commit
You can fix this with 'git rebase --edit-todo' and then run 'git rebase --continue'.
Or you can abort the rebase with 'git rebase --abort'.

Reply:
Found the console, Should I enter the first two in?  
Don't want to touch anything without being sure I wont screw it up worse.

Reply:
> So I did this in fork, Got this error

At the bottom of the list, the first commit shouldn't be "Fixup". Additionally, you should set the 5 commits at the top to "Fixup" as well.

Reply:
So fixup the five I want to keep and squash everything else? Or just squash the first one after and fixup the rest?

Reply:
"Pick" the first one at the bottom and "Fixup" all the rest. You will be able to change the commit message afterwards when you amend this lone commit with the unwanted changes removed.

Reply:
https://friendica.myportal.social/photo/2232433858647e3f3c2bf41157421374-0.png
Did this, got one conflict
https://friendica.myportal.social/photo/1238510374647e3f3c3e0e1647287213-0.png

Reply:
Click on the "merge" button, you will see your version and the `develop` version side by side. Choose either your version or the `develop` version or a mix of both (the bottom window is a text editor), click resolve and you can continue with the rebase.